dashboard/config/linux/bits/base.yml

```diff
@@ -73,7 +73,7 @@ config:
  - USER_NS
  - UTS_NS
  # Depends on CONFIG_SYSVIPC.
- - CONFIG_IPC_NS: [optional]
+ - IPC_NS: [optional]
```

Comment:
Maybe the typo is why we had `[optional]` here?
We tried to enable `CONFIG_CONFIG_IPC_NS` but couldn't and instead of realizing this is a typo I just added `[optional]`.
Please try to remove it and see if everything works without it.

Reply:
```
generating android-5.10-base...
android-5.10 failed:
base.yml:76: IPC_NS is not present in the final config
saved config before olddefconfig to dashboard/config/linux/android-5.10.config.tmp
```
Perhaps this has something to do with CONFIG_SYSVIPC mentioned above?

Reply:
OK, then let's leave [optional].